tests/test_fixtures/rule_based_profiler/example_notebooks/BasicExample_RBP_Instantiation_and_running.ipynb

```diff
@@ -1014,8 +1014,8 @@
     "expect_column_max: DefaultExpectationConfigurationBuilder = DefaultExpectationConfigurationBuilder(\n",
     "    expectation_type=\"expect_column_max_to_be_between\",\n",
     "    column=\"$domain.domain_kwargs.column\", \n",
-    "    min_value=\"$parameter.max_range_parameter_builder.value.value_range[0]\",\n",
-    "    max_value=\"$parameter.max_range_parameter_builder.value.value_range[1]\",\n",
+    "    min_value=\"$parameter.max_range_parameter_builder.value[0]\",\n",
+    "    max_value=\"$parameter.max_range_parameter_builder.value[1]\",\n",
     ")"
```

Comment:
Thank you for this @alexsherstinsky .. it really makes simpler and more elegant. I love it